src/views/create-update-recipe.twig

```diff
@@ -99,6 +98,22 @@
                 $recipeSummaryTextarea.data('caption', 'false');
             }
 
+            $(".clean-on-paste").bind('paste', function (e) {
+                var elem = $(this);
+                setTimeout(function () {                    
+                    var lines = elem.val().split(/\n/);      
```

Comment:
`split` argument shouldn't need to be a regex, right?
I wonder if it'll work on windows as well?

Reply:
we need here regex 
see 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split